packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/method_channel/method_channel_google_maps_flutter.dart

```diff
@@ -364,6 +364,19 @@ class MethodChannelGoogleMapsFlutter extends GoogleMapsFlutterPlatform {
     });
   }
 
+  @override
+  Future<void> animateCameraWithDuration(
+    CameraUpdate cameraUpdate, {
+    required int mapId,
+    Duration? duration,
+  }) {
+    return channel(mapId)
+        .invokeMethod<void>('camera#animateWithDuration', <String, dynamic>{
+      'cameraUpdate': cameraUpdate.toJson(),
+      'duration': duration?.inMilliseconds,
```

Comment:
It would be helpful to include units in the key (`durationInMilliseconds`) so that the code is more self-documenting on the native side.